### DISCUSSION TITLE
how can I contribute to this project?

### DISCUSSION THREAD

**Discussion by ghost** [searx/searx#3434] [General]
Hi everyone.

I want to help, but I want to know if this project is active. *Is this project active? how can I contribute to this project?*

**Comment by unixfox** [comment]
Please see:
- https://github.com/searx/searx/issues/3269
- https://github.com/searx/searx#is-searx-in-maintenance-mode

**Reply by ghost** [reply]
Hi unixfox.

> Please see:
> https://github.com/searx/searx/issues/3269
> https://github.com/searx/searx#is-searx-in-maintenance-mode

thank you for feedback.
